spec/omniauth/strategies/auth0_spec.rb

```diff
@@ -123,6 +123,20 @@
       expect(redirect_url).not_to have_query('connection')
     end
 
+    it 'redirects to hosted login page with screen_hint=signup' do
+      get 'auth/auth0?screen_hint=signup'
```

Comment:
There's a existing test for checking that on a "normal authorize call" (without extra args) these keys are not part of the query. Let's add this one there, and also the ones that are missing from previous changes.

The test is on these lines: https://github.com/auth0/omniauth-auth0/blob/894d4e29046fb2944119c2fa462ef6aa4436fd62/spec/omniauth/strategies/auth0_spec.rb#L84-L86

Let's add a line for testing that the following keys are NOT in the query:
- connection_scope
- screen_hint